CHANGES.rst

```diff
@@ -9042,6 +9609,23 @@ astropy.coordinates
 - The ``__repr__`` of coordinate objects now shows scalar coordinates in the
   same format as vector coordinates. [#3350, 3448]
 
+- Refactor conversions between ``GCRS`` and ``CIRS,TETE`` for better accuracy
```

Comment:
Why are these here?

Reply:
I'm not sure but I bet it was due to rebase getting very confused?  Checking now, thanks for catching that.

Reply:
No, never mind, this was a my-bad in e92f9cf7c92db8e0d18b5404cc387e458daedf3b due to getting lost in the changelog.  Will fix in a commit momentarily

Reply:
OK should be fixed now.  Thanks for catching that @bsipocz !